db/migrate/20240314230055_create_extensions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateExtensions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :extensions do |t|
+      t.references :assignment, foreign_key: true
+      t.string :student_email
+      t.datetime :initial_due_date
+      t.datetime :new_due_date
+      t.bigint :last_processed_by_user_id
```

Comment:
[Slack Message Chain](https://berkeley-cs169.slack.com/archives/C06GQ5AJG1J/p1710923526249849?thread_ts=1710903303.007989&cid=C06GQ5AJG1J).

---

Since this does directly reference another user that has to be in the db already, its good to leverage the power of RDBMSs with relations as opposed to having a bigint here and forcing a join to get the rest of the provisioning user's information.